src/GafferImage/Warp.cpp

```diff
@@ -602,6 +609,14 @@ void Warp::hashChannelData( const GafferImage::ImagePlug *parent, const Gaffer::
 IECore::ConstFloatVectorDataPtr Warp::computeChannelData( const std::string &channelName, const Imath::V2i &tileOrigin, const Gaffer::Context *context, const ImagePlug *parent ) const
 {
 	ConstCompoundObjectPtr sampleRegions = sampleRegionsPlug()->getValue();
+
+	{
+		Gaffer::ContextPtr sampleRegionsContext = new Gaffer::Context( *context, Gaffer::Context::Borrowed );
+		sampleRegionsContext->remove( ImagePlug::channelNameContextName );
+		Gaffer::Context::Scope sampleRegionsScope( sampleRegionsContext.get() );
+		sampleRegions = sampleRegionsPlug()->getValue();
```

Comment:
Aren't we calling `getValue()` twice now, once in the normal context and once in the global context?